### PR TITLE
Initialize CPA web app

### PR DESCRIPTION
**Pull request by itsmostafa**
# 

## 🗣 Description

Initialize the controller web application

## 💭 Motivation and Context

To get started on the web controller

## 🧪 Testing

None yet

## 🚥 Types of Changes

Initialize code

### CODE REVIEW

File: controller/src/config/settings.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    },
+]
+
+WSGI_APPLICATION = 'config.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/3.0/ref/settings/#databases
+
+DATABASES = {
```
Comment (xlyk) [inline controller/src/config/settings.py]: do you require a database?

File: controller/src/config/tasks.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# setup celery
+celery = Celery('tasks', broker=os.environ.get("RABBITMQ_URL"))
+
+# configure celery
+celery.conf.update(
+    task_serializer='json',
+    accept_content=['json'],
+    result_serializer='json',
+    timezone='America/Chicago',
```
Comment (xlyk) [inline controller/src/config/tasks.py]: here the timezone is 'America/Chicago', but in settings.py its set to UTC. They should probably be the same

**Review by xlyk — CHANGES_REQUESTED** [review]

File: controller/src/config/settings.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    },
+]
+
+
+# Internationalization
+# https://docs.djangoproject.com/en/3.0/topics/i18n/
+
+LANGUAGE_CODE = 'en-us'
+
+TIME_ZONE = 'America/Chicago'
```
Comment (pmercado23) [inline controller/src/config/settings.py]: ToDo: we should conform weather we are using UTC or another timezone for this app.

Comment (itsmostafa) [reply controller/src/config/settings.py]: Good point. I think its best to switch to UTC for now and we can save that for a discussion

**Review by pmercado23 — APPROVED** [review]
good base start, might be worth asking about UTC time and if we want to parse out some configs into a json for easier dev/deployment

File: controller/src/config/settings.py (hunk 3 above)
Comment (pmercado23) [reply controller/src/config/settings.py]: I can see reason for either, having in UTC is the standard but assuming we are using it only for US clients then we can use EST timezone too.

**Review by pmercado23 — APPROVED** [review]
LGTM :shipit:

**Pull request merged**